docs/installation/installation.rst

```diff
@@ -309,14 +346,10 @@ either of the following methods:
       To install WSL, users can follow the instructions from 
       `Microsoft <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
       More comprehensive instructions can be found
-      `here <https://www.windowscentral.com/install-windows-subsystem-linux-windows-10>`_.
+      `here <https://www.windowscentral.com/install-windows-subsystem-linux-windows-10>`__.
```

Comment:
This same error happened to me before where something (I think GitHub) added a double underscore at the end of the links.

Reply:
I think I added the double underscore at the end to convert it an anonymous link to prevent the duplicated link keyword error in Sphinx as we are using `here` as a link keyword multiple times in the same `.rst`. Do you happen to know of another way to prevent the duplication error in Sphinx?

Reply:
Oh I didn't know that was a feature! I thought it was an error because it happened to my code automatically on one of my recent commits.

Reply:
That's cool though that sth does that for automatically!